pytest.ini

```diff
@@ -1,2 +1,4 @@
 [pytest]
 testpaths = cumulusci
+filterwarnings =
+    ignore::DeprecationWarning
```

Comment:
@Br4nd0R I know these are coming from libraries we use instead of from our own code, but I don't like ignoring all deprecation warnings since we could miss something in the future.

Reply:
apologies, forgot to remove this. The warnings were cluttering the debug output in VSCode.